invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LoanDetails/components/LoanMetadata/LoanMetadata.js

```diff
@@ -98,7 +98,7 @@ export default class LoanMetadata extends Component {
       },
       {
         name: 'Delivery',
-        value: data.metadata.delivery.method,
+        value: data.metadata.delivery ? data.metadata.delivery.method : null,
```

Comment:
Maybe an empty string instead of `null`, I am not sure what is preferable.

Reply:
Or hide Delivery completely if there is no delivery method.